include/rmm/mr/device/cuda_async_memory_resource.hpp

```diff
@@ -57,10 +68,14 @@ class cuda_async_memory_resource final : public device_memory_resource {
    * initial pool size is half of the available GPU memory.
    * @param release_threshold Optional release threshold size in bytes of the pool. If no value is
    * provided, the release threshold is set to the total amount of memory on the current device.
+   * @param export_handle_type Optional `cudaMemAllocationHandleType` that allocations from this
+   * resource should support for interprocess communication (IPC). Default is
```

Comment:
```suggestion
   * resource should support interprocess communication (IPC). Default is
```

Reply:
Please read again. "For" is not correct here. :)

Reply:
Right think Vyas is suggesting removing it

Reply:
Yup there is an extraneous "for" already present here that I would like to remove. I am agreeing with you @harrism :)